README.md

```diff
@@ -24,14 +24,22 @@ sum_array
 */
 ```
 
-# Options
+# Configuration
 
-There are two `postgresql.conf` settings that must be configured:
+First, plrust must be configured as a `shared_preload_libraries` entry in `postgresql.conf`.  For example:
+
+```
+shared_preload_libraries = 'plrust'
+```
+
+Failure to do so will cause unexpected behavior around the DROP FUNCTION, DROP SCHEMA, and ALTER FUNCTION commands.
```

Comment:
Do you want to add something like `process_shared_preload_libraries_in_progress` check?
https://github.com/pgaudit/pgaudit/blob/master/pgaudit.c

It'll only error during CREATE LANGUAGE/CREATE EXTENSION plrust is called but that seems better than nothing

Reply:
Oh, yes.  That's great.  Didn't know was a thing.